docs/source/best-practices/homequota.rst

```diff
@@ -0,0 +1,83 @@
+*******************************************************
+Home Directory Storage Quota
+*******************************************************
+There are strict quotas for each home directory (i.e.,  ``/home/<username>``) and staying quota is vital in preventing issues using Open OnDemand and performing tasks on the HPC. For more information about the quota see :ref:`discovery_storage`. This page provides some best practices for staying under quota.
```

Comment:
Consider rewording the opening paragraph (perhaps a bit confusing, as is, for it sounds a bit like there is another page about the quota.):

> There are strict quotas for each home directory (i.e.,  ``/home/<username>``), and staying within the quota is vital for preventing issues on the HPC. This page provides some best practices for keeping within quotas.
> 
> For more information about the data storage on the HPC, see :ref:`discovery_storage`.

Feel free to follow your preference—just a thought.